core/domain/user_jobs_one_off.py

```diff
@@ -68,6 +68,29 @@ def reduce(key, version_and_exp_ids):
                     edited_exploration_ids))
 
 
+class UsernameLengthDistributionOneOffJob(jobs.BaseMapReduceJobManager):
+    """One-off job for calculating the distribution of username lengths."""
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [user_models.UserSettingsModel]
+
+    @staticmethod
+    def map(item):
+        if item.username is not None:
+            yield (len(item.username), {
+                'username': item.username
+            })
+
+    @staticmethod
+    def reduce(key, usernames):
+        username_list = [ast.literal_eval(v) for v in usernames]
+        username_length_model = (
```

Comment:
I think you can retrieve the job model from the datastore and check the output that's stored in it. There may be examples of this in core/jobs_test.py, otherwise you might have to dig into the codebase a bit more.